requirements.txt

```diff
@@ -1,3 +1,2 @@
 numpy
-onnxruntime>=1.13

```

Comment:
If you do this, you will have to update the CI tests as well.

Reply:
updated CI tests and added ImportError exception handling to clearly identify for the user how to correctly install the package.